### PR TITLE
Pepcus Website CSS Code Refactoring(10 Dec)

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

File: careers.html [hunk 1]
```diff
@@ -10,16 +10,17 @@
 	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
     <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" crossorigin="anonymous">
     <link rel="stylesheet" href="css/owl.carousel.css">
-    <link rel="stylesheet" type="text/css" href="css/owl.carousel.min.css">
+<!--    <link rel="stylesheet" type="text/css" href="css/owl.carousel.min.css">-->
```
Comment (amitjain24524) [inline careers.html]: please remove commented code.

File: careers.html [hunk 2]
```diff
@@ -10,16 +10,17 @@
...
     <link rel="stylesheet" href="css/owl.theme.css">
     <link rel="stylesheet" type="text/css" href="css/jquery.fancybox.min.css">
     <link rel="stylesheet" href="css/jquery.fancybox-transitions.css">
     <link rel="stylesheet" type="text/css" href="css/jquery.mosaic.min.css">
-    <link rel="stylesheet" href="css/demo.min.css">
+<!--    <link rel="stylesheet" href="css/demo.min.css">-->
```
Comment (amitjain24524) [inline careers.html]: same

File: careers.html [hunk 3]
```diff
@@ -10,16 +10,17 @@
...
     <link rel="stylesheet" href="css/demo.css">
-    <link rel="stylesheet" type="text/css" href="css/careers.css">
+<!--    <link rel="stylesheet" type="text/css" href="css/careers.css">-->
```
Comment (amitjain24524) [inline careers.html]: same

File: index.html [hunk 4]
```diff
@@ -13,7 +13,7 @@
     <title>Pepcus Software Services</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
     <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" crossorigin="anonymous">
-    <link rel="stylesheet" href="css/animate.css">
+<!--    <link rel="stylesheet" href="css/animate.css">-->
```
Comment (amitjain24524) [inline index.html]: same

File: careers.html (hunk 3 above)
Comment (Palakagrawal28) [reply careers.html]: Done

File: careers.html (hunk 2 above)
Comment (Palakagrawal28) [reply careers.html]: Done

**Pull request merged**